recipes/bandage/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Run qmake to generate a Makefile
+qmake Bandage.pro
+
+# fix the makefile
+sed -i 's/isystem/I/' Makefile
```

Comment:
According to [this](https://stackoverflow.com/questions/9329772/why-would-sed-not-work-in-my-bash-script) if you replace 
```
sed -i 's/isystem/I/' Makefile
```
by 
```
sed -i "s/isystem/I/" Makefile
```

Recipes can build on osx

Reply:
According to [here (in the comments)](https://stackoverflow.com/a/9329869/6914989) osx would require `sed -ie "s/isystem/I/" Makefile`.
Details [there](https://stackoverflow.com/questions/2320564/i-need-my-sed-i-command-for-in-place-editing-to-work-with-both-gnu-sed-and-bsd) suggest that it will create a backup file that could require cleaning.